src/app/modules/jobs/components/jobs-panel/jobs-panel.component.spec.ts

```diff
@@ -1,76 +1,89 @@
 import { HarnessLoader } from '@angular/cdk/testing';
 import { TestbedHarnessEnvironment } from '@angular/cdk/testing/testbed';
 import { MatButtonHarness } from '@angular/material/button/testing';
-import { MatDialog, MatDialogRef, MAT_DIALOG_DATA } from '@angular/material/dialog';
+import { MatDialogRef } from '@angular/material/dialog';
 import { Router } from '@angular/router';
-import {
-  createRoutingFactory, mockProvider, Spectator,
-} from '@ngneat/spectator/jest';
+import { Spectator, createRoutingFactory, mockProvider } from '@ngneat/spectator';
```

Comment:
Change to '@ngneat/spectator/jest'

Reply:
I stuck for a day with this. Thank you 🙏🏻